scripting/2.0/modules/ROOT/pages/scripting-module-studio.adoc

```diff
@@ -171,19 +171,18 @@ In the *Configuration XML* editor, place the script in between `<scripting:code>
 You can configure other additional fields for the Execute operation, such as Parameters, Target Variable, and Target Value.
 
 
-=== Parameters
+=== Parameters Parameter
 
-[%header,cols="20s,20a,35a,20a,5a"]
-|===
-| Name | Type | Description | Default Value | Required
-| Parameters | Map<String, Object> | Variables provided to the script as bindings | |
-|===
-
-In the *Parameters* box of the *General* tab, define input parameter values for the script to use through DataWeave. For the DataWeave expression to work correctly, you need to combine these parameters' types output, where keys are strings and values are any object. +
+Use the *Parameters* parameter to define input values for the script to use through DataWeave. For the DataWeave expression to work correctly, you need to combine these parameters' types output, where keys are strings and values are any object. +
```

Comment:
"need to" > "must" (this is shorter)

"parameters' types output" is awkward. I suggest "You must combine the output types of the parameters," if this has the same meaning. I would also change "where" to "in which," because "where" usually implies a location. Kim posted this in para-docs about the two - your use of "where" might be OK according to this, but I am not sure.

Is it okay to use "where" instead of that stuffy ol' "in which"?
Thanks for asking!
I just had an interesting convo with a writer about the wording of some Fixed Issues list items in release notes, many of which began with "Fixed an issue where <this happened>." Seeing "where" used this way is so common that it nearly doesn't bear comment: like using "comprises," it's probably best to reword:
     Fixed <this issue>.
     <This issue> no longer happens.
     In this release, <this solution> happens.
     <This solution> now happens.
(Stay tuned for proposed release notes formatting and boilerplate text!)
The basic difference between using "where" and "in which" is one of casual vs. formal discourse. Our tone recommendation is to be simple, direct, and conversational, but to void being too informal or familiar with our audience.
In some instances, it is not correct to use "where," which typically indicates a place and not a time or situation.  An exception to this is describing a condition or set of conditions that are true.
I just found this useful, but long, article:
https://www.quickanddirtytips.com/education/grammar/where-versus-in-which
The gist of the matter is at the very end:
"Every time you write something, you as the author have to decide which words or phrases are appropriate. So it’s up to you to decide whether to use “which” or “where” in your particular sentence.... be aware of one common error. When you use “where,” you need to ensure that you’re talking about place, not time."
~~~*~~~*~~~*~~~*~~~
As always, please contact me directly if you have questions or comments. (edited)